tests/functional/routes/test_account.py

```diff
@@ -30,7 +30,7 @@ def test_obtain_email_confirm_token(
         )
 
         assert res.status_code == 200
-        send_confirm_email.assert_called_once()
+        assert send_confirm_email.call_count is 2
```

Comment:
Why is this called twice now?

Reply:
The first one is sent when we added an unconfirmed user in lines 18-20

Reply:
The second is from the post request